lambdaHttp/src/main/scala/Handler.scala

```diff
@@ -14,8 +14,8 @@ object Handler {
   def handle[F[_]: Effect](
       endpoints: List[ServerEndpoint[_, F]],
   ): FunctionType = { (event, context) =>
-    println(js.JSON.stringify(event))
-    println(js.JSON.stringify(context))
+    // println(js.JSON.stringify(event))
+    // println(js.JSON.stringify(context))
```

Comment:
I understand that we do not want them in the dev case, but I find them very useful when debugging lambdas

Reply:
We should configure it using ENV variables. What would be a good name?